Userland/Applications/Spreadsheet/Cell.cpp

```diff
@@ -13,7 +13,8 @@ namespace Spreadsheet {
 
 void Cell::set_data(String new_data)
 {
-    if (m_data == new_data)
+    //If we are a formula, we do not save the beginning '=', if the new_data is "" it is different from us.
```

Comment:
Please put a space between the `//` and `If` here.

Reply:
Will do!